lib/SIL/Verifier/SILVerifier.cpp

```diff
@@ -1119,6 +1119,12 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
   void visitSILArgument(SILArgument *arg) {
     CurArgument = arg;
     checkLegalType(arg->getFunction(), arg, nullptr);
+
+    // Ensure flags on the argument are not stale.
+    require(!arg->getFunction()->hasOwnership() ||
```

Comment:
Did you want to add a verifier test for this?

Reply:
Done.